### PR TITLE
Update READMEs

### PR DESCRIPTION
**Pull request by nglgzz**
### Please check if the PR fulfills these requirements
- [x] The commit message follows our [guidelines](https://github.com/neo4j-devtools/relate/blob/master/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

### What kind of change does this PR introduce?
Docs update

### What is the current behavior?
The READMEs don't provide much useful information to new users.

### What is the new behavior?
- The main README describes what Relate is, the main surfaces we expose, and basic concepts. 
- CLI and web READMEs include a guide for getting started and a description of how to find API documentation.

### Does this PR introduce a breaking change?
No

### Other information:
The README for the common package still needs to be revisited, and it would probably make sense to add examples to show how it would work in different scenarios.

When reviewing this PR it might be easier to check the [file view](https://github.com/nglgzz/relate/blob/23968362c449e3535392ea898685c63fe6d4233c/README.md) rather than the diff.

### CODE REVIEW

**Review by oskarhane — APPROVED** [review]
Very nice!

File: README.md [hunk 1]
```diff
@@ -1,52 +1,66 @@
...
+contain multiple databases.
 
-### Generate documentation
+### Project
 
-```
-$ npm run generate:docs
-```
+A project in Relate is a folder with some extra metadata and with possible
+associations to databases or DBMSs.
```
Comment (jk05) [inline README.md:62]: `files`?

**Comment by jk05** [comment]
great stuff. Just an aside - the arguments in the CLI docs, e.g `ENTITYNAMEORID` or `BACKUPIDORPATH`, is it convention to write like this instead of snake case for example?

**Review by jk05 — APPROVED** [review]

**Comment by nglgzz** [comment]
> great stuff. Just an aside - the arguments in the CLI docs, e.g `ENTITYNAMEORID` or `BACKUPIDORPATH`, is it convention to write like this instead of snake case for example?

No, you're right they should be in snake case. These are automatically generated by oclif from our camel case names, so I'm guessing that's why they don't have underscores, but I think there should be a way to override them.

**Pull request merged**